README.md

```diff
@@ -7,6 +7,19 @@ Na gałęzi ```main``` mamy wersję aplikacji, która jest aktualnie na App Stor
 Do odpalenia apki potrzebne są klucze API do Google Maps, bez nich nie wyświetli wam się mapka w aplikacji. 
 Ze względów bezpieczeństwa nie udostępniamy ich publicznie w repozytorium, aby je dostać wystarczy poprosić na Discordzie. Po uzyskaniu klucza należy:
 
+### Automatyczne dodanie klucza
+W Git Bash należy przejść do katalogu ze sklonowanym repozytorium (katalog z folderem `.git`) i wywołać komendę:
```

Comment:
Jeżeli mamy zainstalowanego Git Bash, to nie musimy odpalać tej komendy z tej aplikacji, można to zrobić za pomocą CMD, PowerShell albo Windows Terminal.

Ta podpowiedź, że to katalog z `.git` może być mało pomocna, bo ten katalog jest ukryty :P.

Reply:
Bez WSL skrypty basha działają tylko w Git Bash. Nie mam pomysłu jak inaczej podopowiedzieć, który to katalog repozytorium.